### DISCUSSION TITLE
Errors Running Install Script

### DISCUSSION THREAD

**Discussion by jameswinchester660** [phoenixbyrd/Termux_XFCE#51] [Q&A]
Hello,

I keep getting repo errors while running the install script. After choosing my username and picking the main (and also mirror) repo, I get a string of not found errors and then the script terminats.

EDIT: Running on a Samsung S24 using latest update.

Thanks, 

James 
![Screenshot_20240601_115124_Termux](https://github.com/phoenixbyrd/Termux_XFCE/assets/76274160/a1ec173b-3fdf-48d0-b8ca-4289adaf2ee6)

**Comment by jameswinchester660** [comment]
Edit 2: The repositories error happens when I try to run pkg update/upgrade and apk update/upgrade as well

**Comment by NGC-1512** [comment]
Maybe this answer Will help:
https://github.com/termux/termux-app/discussions/2604#discussioncomment-2223489
